src/com/projectkorra/projectkorra/firebending/BlazeArc.java

```diff
@@ -39,7 +31,12 @@ public BlazeArc(final Player player, final Location location, final Vector direc
 		super(player);
 		this.range = this.getDayFactor(range);
 		this.speed = getConfig().getLong("Abilities.Fire.Blaze.Speed");
-		this.interval = (long) (1000. / this.speed);
+		this.interval = (long) (1000.0 / this.speed);
+
+		if(bPlayer.canUseSubElement(SubElement.BLUE_FIRE)) {
+			this.range += BlueFireAbility.getRangeFactor() * range - range;
```

Comment:
Shouldn't this just be `this.range *= BlueFireAbility.getRangeFactor();`

Reply:
Nay

Reply:
Ok